unixfs/mod/dagmodifier.go

```diff
@@ -19,8 +21,12 @@ import (
 	ipld "gx/ipfs/Qme5bWv7wtjUNGsK2BNGVUFPKiuxWrsqrtvYwCLRw8YFES/go-ipld-format"
 )
 
-var ErrSeekFail = errors.New("failed to seek properly")
-var ErrUnrecognizedWhence = errors.New("unrecognized whence")
+// Common errors
+var (
+	ErrSeekFail           = errors.New("failed to seek properly")
+	ErrUnrecognizedWhence = errors.New("unrecognized whence")
+	ErrNotUnixfs          = fmt.Errorf("dagmodifier only supports unixfs nodes (proto or raw)")
```

Comment:
Might as well be consistent and use the `errors` package.

Reply:
out of scope I'd say

Reply:
ah, yeah, got it now